src/components/confirm-dialog/ConfirmDialog.tsx

```diff
@@ -48,12 +50,25 @@ const ConfirmDialog: FC<ConfirmDialogProps> = ({
         <Typography>{t(message)}</Typography>
       </DialogContent>
       <DialogActions sx={styles.actions}>
-        <AppButton onClick={onConfirm}>
-          {confirmButton ? confirmButton : t('common.yes')}
-        </AppButton>
-        <AppButton onClick={onDismiss} variant={ButtonVariantEnum.Tonal}>
-          {cancelButton ? cancelButton : t('common.no')}
-        </AppButton>
+        {revertButtons ? (
+          <>
+            <AppButton onClick={onDismiss} variant={ButtonVariantEnum.Tonal}>
```

Comment:
As I see, the buttons are the same, you are just changing their places, try to use for example flexDirection property for reversing them, and pass revertButton to styles for applying appropriate direction